bin/feature

```diff
@@ -199,6 +199,9 @@ when 'merge'
    exit 1 unless confirm("Merge feature branch named: '#{feature}' ?")
 
    update = Git::submodules_update("get")
+   pull_desc = pull_info[:description].shellescape
+   failure_message = highlight("Merge conflicts detected, merge #{dev_branch} into #{feature} and resolve conflicts.".shellescape)
```

Comment:
That's what `feature merge` does. When it *can't* it tells you to do the opposite (merge master into your_branch) so the conflict resolution get's resolved before the branch is merged into master.